webapp/src/main/java/org/opentestsystem/rdw/admin/repository/impl/JdbcGroupRepository.java

```diff
@@ -79,6 +99,13 @@ private static Group mapResultToGroup(final ResultSet row) throws SQLException {
                 .build();
     }
 
+    private static Map<String, Object> getGroupParameters(final Group group) {
+        return ImmutableMap.<String, Object>builder()
+                .put("id", group.getId())
+                .put("deleted", group.getIsDeleted())
```

Comment:
I'm only implementing the the `deleted` field at this point.  When we come later (in phase 2?) to implement Edit, we can just include the other group fields once they've been validated.